content/docs/faqs.md

```diff
@@ -0,0 +1,76 @@
+---
+title: FAQs
+weight: 100
+description: Check out frequently asked questions about agentgateway. 
+--- 
+
+Check out frequently asked questions about agentgateway. 
+
+## What are MCP and A2A? 
+
+With agentic artificial intelligence (AI) changing the way organizations build and deliver applications, organizations face the challenge of rapidly adopting new technologies and interoperability protocols to connect agents and tools in fragmented environments. To accelerate agent development, infrastructure is needed that transcends the rapidly changing landscape.
+
+[Model Context Protocol (MCP)](https://modelcontextprotocol.io/docs/getting-started/intro) has become the de facto standard for retrieving and exchanging context with Large Language Models (LLMs) and for connecting LLMs to tools. [Agent-to-Agent (A2A)](https://github.com/a2aproject/A2A) is a newly introduced complementary protocol to MCP that solves for long-running tasks and state management across multiple agents. MCP and A2A are both JSON-RPC protocols that define the structure of how an agent describes what it wants to do, how it calls tools, and how it hands off tasks to other agents.
+
+## Why do I need a gateway for MCP and A2A communication?
+	
+Gateways provide a consistent way to manage MCP and A2A communication across multiple tools and agents. While MCP and A2A define the RPC communication protocol for agents and tools, they currently do not address enterprise-level concerns regarding authentication, authorization, resiliency, tracing, tenancy, and guardrails. In addition, LLMs, agents, and tools are typically spread across remote networks, which requires low latency and handling of retries, timeouts, and failures. 
+
```

Comment:
we should highlight the complexity of agent communication here, something like: Traditional API gateways and reverse proxies aren’t built for MCP or A2A. They operate with a simple model: one request in, pick a backend endpoint, one request out, with no session context or ongoing connection state. AI agents, whether agent-to-agent or agent-to-tool, maintain long-lived sessions, and every request is tied to that session context.